lib/client.js

```diff
@@ -42,11 +42,12 @@ function getErrorMessages(responseBody) {
   }
 }
 class Client {
-  constructor(apiKey, baseUrl) {
+  constructor(apiKey, baseUrl, timeout) {
```

Comment:
Thank you so much for making this change. This is great. Could you change this to be an optional config object? That just future proofs this method in case we have additional configuration needs.

```suggestion
  constructor(apiKey, baseUrl, { timeout } = {}) {
```

Then later:
```js
this.requestConfig = { timeout };
```

And finally:
```js
_jsonRequest(method, path, params) {
    return this._wrap({
      method: method,
      url: `${this.baseUrl}${path}`,
      json: params,
      auth: {
        bearer: this.apiKey
      },
      headers: {
        'X-Swiftype-Client': this.clientName,
        'X-Swiftype-Client-Version': this.clientVersion,
        'Content-Type': 'application/json'
      },
      { ...this.requestConfig }
    })
  }
```